sdk/aqueduct/api_client.py

```diff
@@ -230,14 +230,16 @@ def _validate_server_version(self, server_version: str) -> None:
         if parse_version(server_version) > parse_version(sdk_version):
             raise ClientValidationError(
                 "The SDK is outdated, it is using version %s, while the server is of version %s. "
-                "Please update your `aqueduct-sdk` package to the appropriate version. If running "
-                "within a Jupyter notebook, remember to restart the kernel."
+                "Please update your `aqueduct-sdk` package to the appropriate version by running "
+                "`pip3 install aqueduct-sdk==<version>`. If running within a Jupyter notebook, "
+                "remember to restart the kernel."
                 % (sdk_version, server_version)
             )
         elif parse_version(server_version) < parse_version(sdk_version):
             raise ClientValidationError(
                 "The server is outdated, it is using version %s, while the sdk is of version %s. "
-                "Please update your server, or downgrade your SDK so that the versions match."
+                "Please update your server, or downgrade your SDK so that the versions match. "
+                "The guide for updating the server is here: https://docs.aqueducthq.com/guides/updating-aqueduct."
```

Comment:
how does it look in a notebook? Will the clickable part exclude the dot?

Reply:
oh you're right, the period messes up the URL. Fixed.